vue-components/stories/Lookup.stories.ts

```diff
@@ -67,6 +67,24 @@ const vegetableItems = [
 	},
 ];
 
+const addOnScroll = [
```

Comment:
This variable name sounds a bit confusing to me (more like an action than a list). How about `fruitItems`, by analogy with `vegetableItems`? (Are sweet potatoes fruit? :sweat_smile:)

Reply:
I don't think sweet potatoes are a fruit. I added this name because I think this makes it clearer as to what is the purpose of this variable. You are right about adding the instructions, but for me pressing `a` gives enough items to make the test. There are a total of 11 items with the letter `a`, the original menuItem count with `a` is 8 and then it increases to 11

Reply:
I am not sure where to add these instructions, tho... I don't see any other Component with instructions.

Reply:
> but for me pressing a gives enough items to make the test

Yes, for me too, that’s why I think it should be in the instructions ^^

Reply:
Ah, then I misunderstood. I thought you were not getting extra results when testing with the letters a and e.

Reply:
No, it works for me with a, but my first attempt was e :) (and then a bunch of random letters)

Reply:
I still think this variable name is confusing, though. How about `itemsToAddOnScroll`?

Reply:
sure. that sounds like a good name. :)